core/server/master/src/main/java/alluxio/master/AlluxioSimpleMasterProcess.java

```diff
@@ -159,113 +108,30 @@ public void start() throws Exception {
    */
   @Override
   public void stop() throws Exception {
-    stopRejectingServers();
-    if (isGrpcServing()) {
-      stopServing();
-    }
+    mServices.forEach(SimpleService::stop);
```

Comment:
a sanity check that each service should print something on start/stop/demote/promote, except for empty methods.